### PR TITLE
Kathan vakharia-Print occurence of a number in a list/array using binary search technique

### PR DESCRIPTION
**Pull request by Kathan-Vakharia**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added python code for issue #414 

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by Kathan-Vakharia** [comment]
@plazzy99 , @Abhijit2505 ,   Kindly review my PR :)

File: Python/miscellaneous/find_freq_using_bs.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    return count
+
+
+# driver code
+if __name__ == "__main__":
+    N = int(input())
+    arr = list(map(int, input().split()))
+    key = int(input())
+
+    print(f"Ocurrence of '{key}': {get_freq_using_bs(arr, N, key)}")
```
Comment (Kajol-Kumari) [inline Python/miscellaneous/find_freq_using_bs.py]: @Kathan-Vakharia  Please replace this line here by `print("Ocurrence of",key,"is :", get_freq_using_bs(arr, N, key))`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Kathan-Vakharia Please incorporate the requested changes and add a sample input and output below the code section in the commented from.

**Comment by Kathan-Vakharia** [comment]
@Kajol-Kumari , I have made the respective changes kindly review them :).
And ya I did some mistake while squashing the commits because it was my first time 😕 and that's the reason it is showing changes of previous commits also but nothing is broken I believe. Will take care next time 😀

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari , I have made the respective changes kindly review them :).
> And ya I did some mistake while squashing the commits because it was my first time  and that's the reason it is showing changes of previous commits also but nothing is broken I believe. Will take care next time

Hey @Kathan-Vakharia please remove the extra files that got added through your commit, otherwise your PR won't be merged.

**Comment by Kathan-Vakharia** [comment]
@Kajol-Kumari , I have removed those files. Kindly review it again :)
Pardon for taking too long as it is like a new world for me :|

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
The code you have written is doing a partial binary search so it is not accepted
your algo should should be purely binary search

**Comment by Kathan-Vakharia** [comment]
@plazzy99  ,I have updated my code. Kindly review it :)

**Review by vatsalkesarwani12 — APPROVED** [review]
Nice documentation done 🥇 

Keep up the good work

**Comment by Kathan-Vakharia** [comment]
Thanks, means a lot :) , @Kajol-Kumari  and @Abhijit2505 kindly review my pull request as per your convenience  😀

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :)

**Review by Abhijit2505 — APPROVED** [review]
Great 😊

**Comment by iamrajiv** [comment]
@all-contributors please add @Kathan-Vakharia for code

**Pull request merged**